src/main/java/uk/gov/hmcts/ccd/v2/external/controller/CaseAssignedUserRolesController.java

```diff
@@ -228,6 +229,7 @@ public ResponseEntity<CaseAssignedUserRolesResponse> removeCaseUserRoles(
         targetIdamId = "T(uk.gov.hmcts.ccd.v2.external.controller.CaseAssignedUserRolesController)"
             + ".buildOptionalIds(#optionalUserIds)"
     )
+    @Deprecated(forRemoval = true)
```

Comment:
We'll need to review (at a later date) any consumers after releasing this, and advise them to move away to avoid potential issues.

Reply:
deprecated descriptions added to swagger and javadoc in 0bafb53e2e68dfb1a645083a25f66bd291764681